### PR TITLE
Added more docs for query search

### PR DESCRIPTION
**Pull request by marciogualtieri-polyswarm**
Adding more examples to readme files on query searches.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -201,6 +292,49 @@ File 131f95c51cc819465fa1797f6ccacf9d494aaaff46fa3eac73ae63ffbdfd8267
 	Observed filenames: 131f95c51cc819465fa1797f6ccacf9d494aaaff46fa3eac73ae63ffbdfd8267,eicar.com.txt,eicar.txt
 ```
 
+```bash
+$ polyswarm_api -o /tmp/test.txt search search metadata '{ "query": { "exists": {"field": "lief.libraries"}}}'
```
Comment (mrtizmoatwork) [inline README.md]: should there be 2 "search" words in a row?

File: README.md [hunk 2]
```diff
@@ -201,6 +292,49 @@ File 131f95c51cc819465fa1797f6ccacf9d494aaaff46fa3eac73ae63ffbdfd8267
...
+$ cat /tmp/test.txt | more
+Found 1000 matches to the search query.
+Search results for namespace(query=namespace(exists=namespace(field='lief.libraries')))
+File 3a56238678560ca57c87cc08d0e912a7cfddaa65c449135d908590780a086232
+	File type: mimetype: application/x-dosexec, extended_info: PE32 executable (GUI) Intel 80386, for MS Windows, UPX compressed
+	SHA256: 3a56238678560ca57c87cc08d0e912a7cfddaa65c449135d908590780a086232
+	SHA1: 29629198ac945de358952337da1015a8252c11f1
+	MD5: e74b9320e6c15363fa30319af75ff3e5
+	First seen: Thu, 16 May 2019 14:26:26 GMT
+	Observed countries: ,PR
```
Comment (mrtizmoatwork) [inline README.md]: the output of this value has a problem with the first comma ",". It should be omitted when there is only one value.

File: README.md (hunk 1 above)
Comment (marciogualtieri-polyswarm) [reply README.md]: I don't know how I put this in there. Will fix.

File: README.md (hunk 2 above)
Comment (marciogualtieri-polyswarm) [reply README.md]: Will investigate.

Comment (marciogualtieri-polyswarm) [reply README.md]: Fixed a bug in the output.

**Comment by marciogualtieri-polyswarm** [comment]
> the documentation in README.md looks good except for 2 minor things. Please also add the relevant info to API.md for the updated search endpoint API.

Thanks for the review, Nick. I fixed the typo, the bug in the output, and updated API.md.

**Comment by mrtizmoatwork** [comment]
@supernothing do you have any additional documentation that you want to add? Could add it to this PR.

**Comment by marciogualtieri-polyswarm** [comment]
> looks good to me now.

Thanks, Nick. Do you want a review from @supernothing as well or should I merge it now?

**Comment by marciogualtieri-polyswarm** [comment]
> @supernothing do you have any additional documentation that you want to add? Could add it to this PR.

Cool, will wait until Ben have a look.

**Comment by supernothing** [comment]
I re-arranged some stuff that I had been meaning to, but otherwise looks good. @mrtizmoatwork final OK?

**Pull request merged**